ds_judgements_public_ui/sass/includes/_judgment_text_toolbar.scss

```diff
@@ -76,10 +76,15 @@
     margin-top: $space-2;
     margin-bottom: $space-4;
 
-    font-size: $typography-xl-text-size;
+    font-size: 1.8rem;
```

Comment:
We should ideally be using the size variables wherever we can; is `$typography-lg-text-size` too small for this case?

Reply:
Either let's update `$typography-xl-text-size` to be `1.8rem`, or use `$typography-xl-text-size` here (which is `2rem`) or shift our font sizing if we must have `1.8rem` and `2rem`. CC @Terry-Price what do you think WRT the different font sizes?

Reply:
@Terry-Price please update us then I will change accordingly as this was specific in style guide

Reply:
updated to 2rem using $typography-xl-text-size @jlhdxw @Terry-Price 
![image](https://github.com/user-attachments/assets/f30839f7-a2dd-4373-a55c-39df6cbb2e69)

Reply:
Looks fine to me